packages/react/src/provider/context.ts

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ReactFlagEvaluationOptions, getDefaultedOptions } from '../common/options';
+import { Client } from '@openfeature/web-sdk';
+
+// DO NOT EXPORT PUBLICLY
+export const Context = React.createContext<{ client: Client; options: ReactFlagEvaluationOptions } | undefined>(undefined);
+
+/**
+ * Get a copy of the effective options used for this OpenFeatureProvider.
+ * DO NOT EXPORT PUBLICLY
+ * @returns {ReactFlagEvaluationOptions} options the defaulted options
+ */
+export function useProviderOptions() {
```

Comment:
I didn't think this was a good one to export.

Reply:
I think there are some packages and rollup plugins etc that do this as a post compilation step, but in my experience they are tough to maintain.